front-end/app/checkout.component/payment-processing.component.ts

```diff
@@ -43,6 +43,7 @@ export class PaymentProcess implements OnInit, OnDestroy {
                                     email: buyerDetails.email,
                                     phone: buyerDetails.phone
                                 },
+                                note: buyerDetails.note,
```

Comment:
I didn't have any idea about the note and where it should to be, so I visit the old website of the client and I saw note input in his checkout page and placeholder of the input is this : "Notes about your order, e.g. special notes for delivery."
therefore I added in the checkout page. 
Shall I put it in the basket page now?